lib/KnormRelations.js

```diff
@@ -363,10 +363,11 @@ class KnormRelations {
       }
 
       parseRows(rows) {
+        this.parsedRows = new Map();
```

Comment:
let's move this to `prepareJoins`, somewhere near line 281. then that check you added on line 314 might not be necessary